content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -26,14 +26,39 @@ The original Euclidean Algorithm computes $\gcd(a,b)$ and looks like this:
 
 ```cpp
 ll euclid(ll a, ll b) {
-	for (;b;swap(a,b)) {
-		ll k = a/b;
-		a -= k*b;
+	for (; b; swap(a, b)) {
+		ll k = a / b;
+		a -= k * b;
 	}
-	return a; // gcd(a,b)
+	return a; // gcd(a , b)
 }
 ```
 
+</CPPSection>
+<PySection>
+
+```py
+def euclid(a: int, b: int) -> int:
+    """
+    Returns the GCD of two positive, non-zero numeric values.
+    """
+
+    assert int(a) > 0 and int(b) > 0, \
+        "Arguments must be positive, non-zero numeric values."
+
+    while b > 0: # run Euclidean algo
```

Comment:
```suggestion
    while b > 0:
```

Reply:
Just trying to keep it close to the c++ version.  They have a comment when the algo portion begins, so that's why I put it there.

Is there a USACO Python standards reference I could use for future updates?  I feel like a lot of these changes could have been avoided if that were the case.